src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,33 @@
+import { NavLink } from 'react-router-dom';
+
+import styles from './pagination.module.css';
+
+export default function Pagination({
+  itemsPerPage,
+  totalItems,
+  paginate,
+}: {
+  itemsPerPage: number;
+  totalItems: number;
+  paginate: (pageNumber: number) => void;
+}) {
+  const pageNumbers = [];
```

Comment:
as an option:
`const pageNumbers = Array(Math.ceil(totalItems / itemsPerPage)).fill(null)`

Reply:
If I use this variant, I start getting mistakes "Encountered two children with the same key `null`"